contracts/contracts/levels/DexTwo.sol

```diff
@@ -8,14 +8,13 @@ import 'openzeppelin-contracts-08/access/Ownable.sol';
 contract DexTwo is Ownable {
   address public token1;
   address public token2;
-  constructor() {}
```

Comment:
@GianfrancoBazzani Isn't the point of this preventing the Ownable constructor from running?

Reply:
I think no.

Reply:
I just found out that there is no way to prevent the parent contract's constructor from running in Solidity, which is weird. So, my bad.

Reply:
Well, because of inheritance, the compiler includes all the parent constructors in the deploy bytecode.